packaging/docker/README.md

```diff
@@ -49,33 +49,51 @@ services:
     volumes:
       - /proc:/host/proc:ro
       - /sys:/host/sys:ro
-      - /var/run/docker.sock:/var/run/docker.sock:ro
 ```
 
 ### Docker container names resolution
 
-If you want to have your container names resolved by netdata, you need to do two things:
-1) Make netdata user be part of the group that owns the socket.
-   To achieve that just add environment variable `PGID=[GROUP NUMBER]` to the netdata container,
-   where `[GROUP NUMBER]` is practically the group id of the group assigned to the docker socket, on your host.
-   This group number can be found by running the following (if socket group ownership is docker):
-   ```bash
-   grep docker /etc/group | cut -d ':' -f 3
-   ```
-
-2) Change docker socket access level to read/write like so:
-   from
-   ```
-   /var/run/docker.sock:/var/run/docker.sock:ro
-   ```
-
-   change to
-   ```
-   /var/run/docker.sock:/var/run/docker.sock:rw
-   ```
+There are a few options for resolving container names within netdata. Some methods of doing so will allow root access to your machine from within the container. Please read the following carefully.
+
+#### Docker Socket Proxy (Safest Option)
+
+Deploy a Docker socket proxy that accepts will filter out requests using something like HAProxy so that it restricts connections to read-only access to the CONTAINERS endpoint.
```

Comment:
`Deploy a Docker socket proxy that accepts will filter out requests` -> `Deploy a Docker socket proxy that accepts and filter out requests` unless you meant something else.

Also change HAProxy to [HAProxy](https://docs.netdata.cloud/docs/running-behind-haproxy/) so that it links to our documentation

Reply:
I made this change, but I do worry it might be a little confusing as that documentation doesn't explain how to use HAProxy for this purpose. Though that's a good place for the documentation to be updated to explain it!